### ISSUE TITLE
[CONSOLE] "Smart" setting of decimal places is not that smart

### ISSUE THREAD

**Issue by elvijsTDL** [superfluid-finance/superfluid-console#214]
Really not liking the "Smart" setting in the current state, it changed during merge of GDA and now it shows the "~" in multiple places where it should not have, e.g. if there should be exactly 1400 tokens distributed then it will still show "~" sign , or if the value would be 0 it would still show it as an estimate

Another point is that it is rounding way too aggressive IMO , I would expect "smart" setting to leave a nice trail of decimals depending on the value of the transaction like in the dashboard e.g. 123456789.987654321 would show up in console as "123456789" , and 0.543212345 would end up being something like 0.543 , currently it would aggressively round it up to ~1 , which is a big difference in tokens like ETH and BTC

**Comment by MidnightLightning** [comment]
> Another point is that it is rounding way too aggressive
Agreed; the user experience isn't that great when small fractions are being aggregated together. Browsing the streams for [this project vault](https://console.superfluid.finance/arbitrum-one/accounts/0xf45349435c4A7cbb519987bd4d7460479568fD63?tab=streams), several users are streaming in "~5/month" streams, but they show up as "~0/day". With more precision that would be  "~0.1667/day".

I'd recommend instead of adding more decimal places, instead start to shift up from "day" to "week" to "month" to "quarter" to "year", until you hit a number greater than one. This already seems to be the case in the "Map" view, as the labels on the map for that example address do show those streams as "4.93/Mo".

![image](https://github.com/superfluid-finance/superfluid-console/assets/250250/1b66a897-b50c-4a2d-968a-df9adf3a2ca7)
